src/main/java/com/extrahardmode/config/RootNode.java

```diff
@@ -53,6 +53,22 @@
      */
     PRINT_COMMENTS("Print Comments", VarType.BOOLEAN, true, "If comments like this should be printed"),
     /**
+    * Globally disables certain modules. These settings might improve performance if these modules is not used. 
```

Comment:
Its many months since I had the original plugin on my server so I dont have the timing reports anymore. But they were definitely showing up and it was bugging me for months.

They way I used to have it was that it was disabled for all worlds, but then you did not like that so I tried to use the same code as was used for other things. Was this not want you wanted?